FrontEnd/src/components/profileList/ErrorPage.module.css

```diff
@@ -0,0 +1,68 @@
+.page {
+  width: 1513px;
+  min-height: 800px;
+  display: flex;
+  justify-content: center;
+  background: var(--primary-green-80, #1f9a7c);
+}
+
+.container {
+  display: flex;
+  align-items: center;
+  gap: 80px;
+}
+
+.text404 {
+  display: flex;
+  width: 408px;
+  height: 178px;
+  flex-direction: column;
+  justify-content: center;
+  
+  color: var(--primary-green-20, #effff6);
+  font-feature-settings: "calt" off;
+  font-family: Inter;
+  font-size: 200px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: 20px; /* 10% */
+  letter-spacing: -2px;
+}
+
+.block {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 36px;
+}
+
+.explanation {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+
+  gap: 24px;
+
+  font-family: Inter;
+  font-style: normal;
+  color: var(--primary-green-20, #effff6);
+}
+
+.main {
+  font-size: 48px;
+  font-weight: 700;
+  line-height: 120%;
+  letter-spacing: 0.48px;
+}
+
+.details {
+  width: 548px;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 20px; /* 125% */
+  letter-spacing: -0.16px;
+}
+
+a {
```

Comment:
Even with CSS modules such a selector is very unsafe. Do we want all links to have this font? Even if yes it's better to specify it general style file for the entire app